reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -3781,7 +3781,7 @@ public final T blockLast(Duration timeout) {
 	 *
 	 * @return a {@link Flux} that attempts to continue processing on errors.
 	 */
-	public final Flux<T> errorStrategyContinue(BiConsumer<Throwable, ? super T> errorConsumer) {
+	public final Flux<T> errorStrategyContinue(BiConsumer<Throwable, ? extends T> errorConsumer) {
```

Comment:
Consumer is supposed to be `super` as the callback is in a contravariant position.

Reply:
What @akarnokd said